common/scripts/lib/_parseArgs.sh

```diff
@@ -308,21 +308,20 @@ __prompt_for_inputs() {
 }
 
 __set_host_services() {
-  sed -i '/^HOST_SERVICES/d' "$ADMIRAL_ENV"
-  echo "HOST_SERVICES=true" >> "$ADMIRAL_ENV"
-  source "$ADMIRAL_ENV"
-
-  ## add ssh key to host's authorized keys
-  local authorized_keys="/root/.ssh/authorized_keys"
-  local tmp_authorized_keys="/root/.ssh/tmp"
-  local ssh_public_key=$(cat "$SSH_PUBLIC_KEY")
-  # remove public key from authorized_keys, if present
-  if grep -v "^$ssh_public_key" "$authorized_keys" > "$tmp_authorized_keys"; then
```

Comment:
this causes a problem in centos because the centos file only has the public key admiral adds. grep inverse match (grep -v) will return an error because it doesn't find any other content than the ssh key added by admiral.

adding multiple ssh keys to authorized_keys doesn't cause any error. Even while adding public keys to authorized keys currently during multi-box install, we don't delete the public ssh key if already present. So, this logic of removing ssh key if present is removed.